build/gke-test-cluster/cluster.yml.jinja

```diff
@@ -71,10 +71,7 @@ resources:
                 value: "true"
                 effect: "NO_EXECUTE"
       masterAuth:
-        username: admin
-        password: supersecretpassword
-      legacyAbac:
-        enabled: {{ properties["cluster.legacyAbac"] }}
+        username: ""
```

Comment:
You should remove the masterAuth block. From gcloud `WARNING: Starting in 1.12, new clusters will have basic authentication disabled by default.`

Reply:
> The username to use for HTTP basic authentication to the master endpoint. For clusters v1.6.0 and later, basic authentication can be disabled by leaving username unspecified (or setting it to the empty string).

([link](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.locations.clusters#Cluster.MasterAuth
)) thought it might be good to explicitly turn it off. But removing it will do the same thing, so I'll remove it.

Reply:
I did a test last night and removing the block for master auth in terraform does the right thing in 1.12 (no basic auth, no client cert). It sounds like deployment mgr works the same way, which is nice.